src/views/Home/index.tsx

```diff
@@ -18,10 +18,7 @@ import {
   GetCedarSystemBookmarks,
   GetCedarSystemBookmarks_cedarSystemBookmarks as CedarSystemBookmark
 } from 'queries/types/GetCedarSystemBookmarks';
-import {
-  GetCedarSystems,
-  GetCedarSystems_cedarSystems as CedarSystem
```

Comment:
Nice! Was this line `GetCedarSystems_cedarSystems as CedarSystem` never actually used? I'm trying to understand why it was there previously.

Reply:
It was used in the type assertion I removed: `(systems?.cedarSystems ?? []) as CedarSystem[]`.

Reply:
Ah, thank you! That makes sense.